use-cases/messaging.md

```diff
@@ -0,0 +1,156 @@
+## Messaging
+
+Enable guests to send messages to the enterprise and receive replies, by connecting your chatbot or instant messaging app into the customer messaging system of __Mews Operations__.
+You can do this through the API with the [Messages](../operations/README.md#messages) family of API operations, in conjunction with the `MessageAdded` [General Webhook](../webhooks/wh-general.md).
+
+Message threads or conversations are initiated by the customer and their messages are sent through to the enterprise, where employees are notified and can quickly respond with a message reply added to the thread.
+The typical workflow is as follows:
+
+1. [Customer creates a message within your app](#step-1-customer-creates-a-message-within-your-app)
+2. [Your app creates a new message thread in Mews](#step-2-your-app-creates-a-new-message-thread-in-mews)
+3. [Your app listens for MessageAdded Webhook events](#step-3-your-app-listens-for-messageadded-webhook-events)
+4. [Continue the conversation!](#step-4-continue-the-conversation)
+
+### Step 1: Customer creates a message within your app
+
+The customer or guest creates an instant message within your app, directed to the enterprise.
+
+**Penelope:** ```"Hi! I just wanted to let you know I will be arriving late tonight"```
+
+### Step 2: Your app creates a new message thread in Mews
+
+Your app creates a new message thread in Mews and adds the customer message to the thread.
+
+- Use [Add message thread](../operations/messagethreads.md#add-message-thread) to create a new message thread:
+
+`[PlatformAddress]/api/connector/v1/messageThreads/add`
+
+**Request:**
+
+```json
+	{
+	    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-	D24FB11DBE31D4621C4817E028D9E1D",
+	    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+	    "Client": "SuperMessagingApp 1.0",
+	    "OriginalSender": "Penelope Morgan",
+	    "Subject": "Late arrival"
+	}
+```
+
+**Response:**
+
```

Comment:
Why do we have this duplicity here? We will forget to update it at the moment, we extend the endpoint. I am sure for 100%.

Reply:
I understand, I had the same concern. But I did it anyway, because it is the best way to illustrate how to use the API. Maybe I could add a note to say "examples for illustration purposes only, please consult the Operations section for the detailed implementation". I will sleep on it.